### DISCUSSION TITLE
Yeni katıldım ve yardımcı olmak istiyorum

### DISCUSSION THREAD

**Discussion by TunaKhanate** [Ludeon/RimWorld-Turkish#314] [Q&A]
Yeni katıldım ve çeviriye yardımcı olmak istiyorum ama bu visual studio gibi uygulamaları daha önce kullanmadığım için çevirilerde problem yaşıyorum ve çevirmenler hala aktif mi?

**Comment by TunaKhanate** [comment]
> Yeni katıldım ve çeviriye yardımcı olmak istiyorum ama bu visual studio gibi uygulamaları daha önce kullanmadığım için çevirilerde problem yaşıyorum ve çevirmenler hala aktif mi?

Ayrıca çevirinin asıl oyuna yayınlanıp yayınlanmayacağı kim tarafından kontrol ediliyor?

**Reply by LadySmith** [reply]
Güncellemeden güncellemeye oyun yapımcısı ekliyor. Visual Studio Code kodu okumanıza yardımcı olmak için var. Bayramdan sonra aktifliğe geri döneceğim. O zaman size yardımcı olabilirim. Yeni ek paket çıktı en azından onları eklemem lazım

**Reply by TunaKhanate** [reply]
Biraz aşina oldum nasıl kullanacağımı öğrendim ve birkaç çeviriyi gönderdim. Ek paketin çevirilerinde de yardımcı olmak isterim.
